nextcloudappstore/core/templates/account/signup.html

```diff
@@ -0,0 +1,42 @@
+{% extends "account/base.html" %}
+{% load socialaccount %}
+
+{% load i18n %}
+
+{% block head_title %}{% trans "Signup" %}{% endblock %}
+
+{% block content %}
+<h1>{% trans "Sign Up" %}</h1>
+
+<p>{% blocktrans %}Already have an account? Then please <a href="{{ login_url }}">log in</a>.{% endblocktrans %}</p>
+
+{% get_providers as socialaccount_providers %}
+
+{% if socialaccount_providers %}
+<p>{% blocktrans with site.name as site_name %}Please sign in with one
+of your existing third party accounts.{% endblocktrans %}</p>
+
+<div class="socialaccount_ballot">
+
+  <ul class="socialaccount_providers">
+    {% include "socialaccount/snippets/provider_list.html" with process="login" %}
+  </ul>
+  <div class="login-or">
+    Or, sign up for a {{ site_name }} account.
+  </div>
+
+</div>
+{% endif %}
+
+<form class="signup" id="signup_form" method="post" action="{% url 'account_signup' %}">
+  {% csrf_token %}
+  <table>
+  {{ form }}
+  </table>
+  {% if redirect_field_value %}
+  <input type="hidden" name="{{ redirect_field_name }}" value="{{ redirect_field_value }}" />
```

Comment:
Yes that could possibly be the case. The value comes directly from request.POST or request.GET.

It is set in https://github.com/pennersr/django-allauth/blob/master/allauth/account/views.py#L207 and get_request_param is defined in https://github.com/pennersr/django-allauth/blob/master/allauth/utils.py#L272.

I'll check django.contrib.auth.views.login to see what they do there. I think remember that that view was hardend against malicous redirects.